docs/python/python-quick-start.md

```diff
@@ -0,0 +1,125 @@
+---
+Order: 1
+Area: python
+TOCTitle: Quick Start
+ContentId: 77828f36-ae45-4887-b25c-34545edd52d3
```

Comment:
This was copied from another file so will need to be updated. Where do I get a new ContentId?

Reply:
this is just a GUID, which you can generate however you want (I used the nwallace.createguid extension per Greg's suggestion 😊)